web/src/components/TaskLogView.vue

```diff
@@ -216,6 +216,8 @@ $task-log-status-tab-height:
 .task-log-records__time {
   width: 120px;
   min-width: 120px;
+  user-select: none;
+  opacity: 0.6;
```

Comment:
This part got suggested; I kept it in, but if this is objectionable, it can Just be removed.